aries-site/src/examples/components/checkbox/CheckBoxDisabledExample.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Box, CheckBox, FormField } from 'grommet';
+
+export const CheckBoxDisabledExample = () => {
+  return (
+    <Box>
+      <FormField label="Label" htmlFor="disabled-checkbox" disabled>
```

Comment:
I think we should add 'name' to both FormField and CheckBox to promote best practice. The name will be needed when used within a form.